app/javascript/lib/feature-flags.js

```diff
@@ -0,0 +1,8 @@
+export function getFlagValue(flagId) {
+  return getAllFlags()[flagId]
+}
+
+export function getAllFlags() {
+  // for now, read it off the home page.  Eventually, this will want to be an API call
+  return JSON.parse($('#feature-flags').attr('value'))
```

Comment:
See related suggestions in `covid19.html.erb` and `index.html.erb`.  Benefits of making Rails variables available to JavaScript in our React code via `window.SCP`:

* Enables easier automated testing via Jest
* Eliminates reliance on jQuery and DOM parsing generally
* More consistent with general pattern for this in our React codebase, e.g. `UserProvider.js`
* Faster and simpler to refactor when we use Node environment to get Rails execution context (SCP-2237)

```suggestion
  return ('SCP' in window) ? window.SCP.featureFlags : {test: true}
```